packages/engine/src/camera/systems/CameraSystem.ts

```diff
@@ -248,6 +252,7 @@ export default async function CameraSystem(world: World): Promise<System> {
           coneDebugHelpers.push(arrow)
           arrow.traverse((obj: Object3D) => {
             obj.layers.set(CameraLayers.Gizmos)
+            obj.layers.enable(CameraLayers.Render)
```

Comment:
my intention with the new camera layers, was that rather than adding the render layer to gizmos, we add gizmos to the camera when we want to render them, this way we dont have to set gizmos as often, and we have a single source of truth (the camera) for if gizmos should be rendered (rather than each gizmo)

Reply:
What about the parent/child transform relationship that the camera object introduces. For example, if I want to give a gizmo static world position in the world I need to convert the world to local position in order to make the gizmo stay at the same place.

Reply:
This is a seperate issue i think, layers just allow for a bitmask to optionally render things by category. It has no bearing on render order or transforms.